src/muya/lib/utils/importMarkdown.js

```diff
@@ -581,9 +581,9 @@ const importRegister = ContentState => {
     if (hasCursor) {
       travel(this.blocks)
     } else {
-      const lastBlock = this.getLastBlock()
-      const key = lastBlock.key
-      const offset = lastBlock.text.length
+      const jumpBlock = setToStartNotEndIfNoCursor ? this.getFirstBlock() : this.getLastBlock()
+      const key = jumpBlock.key
+      const offset = setToStartNotEndIfNoCursor ? 0 : jumpBlock.text.length
```

Comment:
You should add the setting entry to Muya's options (`this.muya.options`) like `spellcheckEnabled`. Then, you can remove the function parameters and pass the initial state in editor.vue#L572.